desktop/hardened-desktop/registry-access-management.md

```diff
@@ -0,0 +1,62 @@
+---
+description: Registry Access Management
+keywords: registry, access, managment
+title: Registry Access Management
+redirect_from: 
+- /docker-hub/registry-access-management/
+---
+
+>Note
+>
+>Registry Access Management is available to Docker Business customers only. 
+
+With Registry Access Management, administrators can ensure that their developers using Docker Desktop only access registries that are allowed. This is done through the Registry Access Management dashboard on Docker Hub. 
+
+Below are some example registries administrators can allow: 
+ - Amazon ECR
```

Comment:
Docker Hub should be the first item on this list. Can we also note that Docker Hub will be enabled by default ?